apple/internal/codesigning_support.bzl

```diff
@@ -325,8 +350,11 @@ def _codesigning_args(
     is_device = platform_prerequisites.platform.is_device
     should_sign_sim_bundles = _should_sign_simulator_bundles(
         config_vars = platform_prerequisites.config_vars,
+        features = platform_prerequisites.features,
         rule_descriptor = rule_descriptor,
     )
+
+    # We need to re-sign imported frameworks
```

Comment:
I think we need to keep signing for imported frameworks, in case of incorrect signing on the existing framework?